src/views/pages/ManagePeople/index.vue

```diff
@@ -93,6 +93,15 @@
             </div>
           </div>
 
+          <div class="md-gutter" v-if="form.category == 'farmer'">
```

Comment:
I think we're not recording payment frequency anymore. My understanding is that it is dropped since the business model changed.